src/python/pants/backend/jvm/tasks/ivy_outdated.py

```diff
@@ -64,6 +65,9 @@ def _is_update_coordinate(self, coordinate):
         return True
 
     def execute(self):
+        deprecated_conditional(
+            lambda: True, removal_version="1.30.0.dev0", entity_description="The outdated goal"
```

Comment:
This reads a little weird at first glance. I suggest:

```suggestion
            lambda: True, removal_version="1.30.0.dev0", entity_description="The `outdated` goal"
```